### DISCUSSION TITLE
How to get rid of numbers and underscore in new file?

### DISCUSSION THREAD

**Discussion by FlashDread** [Anjok07/ultimatevocalremovergui#1332] [General]
Hi,

when creating acapellas etc in UVR5, the software always adds number+Underscore at the start of the new file name, e.g. 123_Ava_Max_Psycho(acapella).mp3

Can I stop this numbering so that they arent even created?
Or is there any software which can delete this part/rename this part in bulk for several files?

Thanks!

**Comment by NLesha** [comment]
Go into settings - additional settings - general process settings, and uncheck Settings Test Mode.
